examples/todos/common/src/commonMain/kotlin/org/reduxkotlin/examples/todos/Actions.kt

```diff
@@ -0,0 +1,5 @@
+package org.reduxkotlin.examples.todos
+
+data class AddTodo(val text: String, val completed: Boolean = false)
```

Comment:
Is there value in making this a sealed class so you can case over all the types?

Reply:
I'm think of this as the minimal example of using the redux API.  There will be another example using sealed classes, and a section in the docs.  The docs are progressive, from minimal to advanced

Reply:
Makes sense to me. I just listened to a talk by Py from Square about making good APIs. He said people will always copy "toy examples" and put that kind of code in production, so keep that in mind even with simple examples